e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -115,13 +115,6 @@ describe("snapshots", () => {
       "license-token-missing-banner-dismissal-timestamp",
       new Date().toISOString(),
     );
-
-    // update the Sample db connection string so it is valid in both CI and locally
-    cy.request("GET", `/api/database/${SAMPLE_DB_ID}`).then((response) => {
-      response.body.details.db =
-        "./plugins/sample-database.db;USER=GUEST;PASSWORD=guest";
-      cy.request("PUT", `/api/database/${SAMPLE_DB_ID}`, response.body);
-    });
   }
```

Comment:
have you tested that snapshots still work locally?

Reply:
Yes.
Everything still works locally.

This is still pointing to the same file.
The difference is whether it uses the absolute path (locally) vs  the relative path.

FWIW, I don't think it's even correct to set it to `"./plugins/sample-database.db;USER=GUEST;PASSWORD=guest"`. If you check the string in your local setup, it has the `file:` prefix.

![image](https://github.com/user-attachments/assets/c6aa4e59-ea9d-41e3-8343-a7e58e52ced3)